lib/backend/k8s/resources/ipam_handle.go

```diff
@@ -89,6 +92,43 @@ func (c ipamHandleClient) toV3(kvpv1 *model.KVPair) *model.KVPair {
 	name := c.parseKey(kvpv1.Key)
 	handle := kvpv1.Key.(model.IPAMHandleKey).HandleID
 	block := kvpv1.Value.(*model.IPAMHandle).Block
+	attrs := kvpv1.Value.(*model.IPAMHandle).Attrs
+
+	// Determine who owns this handle, if possible.
```

Comment:
I need to add support for multiple owners, since a handle can be used for multiple IP assignments.